lms/static/sass/course/courseware/_courseware.scss

```diff
@@ -251,6 +256,134 @@ div.course-wrapper {
         }
       }
     }
+    .proctored-exam-skip-confirm-wrapper {
+      border-left: 5px solid $red;
```

Comment:
The `$baseline` variable is equal to 20. So for this value, let's use `border-left: ($baseline/4) solid red;`.